openapi.yaml

```diff
@@ -171,6 +547,365 @@ components:
           schema:
             $ref: '#/components/schemas/ErrorResult'
   schemas:
+    BookId:
+      type: string
+      description: Book ID
+      example: '26'
+    BookType:
+      type: string
+      enum:
+        - book
+    BookAvailable:
+      type: string
+      nullable: false
+      description: Availability of book (true/false).
+      example: true
+    MemberId:
+      type: string
+      description: Member ID
+      example: '001'
+    MemberType:
+      type: string
+      enum:
+        - member
+    MembershipStatus:
+      type: string
+      nullable: false
+      description: Membership status of the member (active/suspended/expired).
```

Comment:
Instead of listing them in `description` filter, you should put them in an `enum` list.

Reply:
done